doc/nrf/app_dev/device_guides/coprocessors/ug_hpf_softperipherals_comparison.rst

```diff
@@ -0,0 +1,116 @@
+     - Allows for the creation of specialized or custom peripherals.
+   * - Use cases
+     - When an application requires an extra or absent peripheral and a corresponding Soft Peripheral is available.
+     - When custom peripheral development or a high-level control is needed.
+   * - Suitability
+     - Suitable for applications built from pre-compiled sources.
+     - Ideal for applications where the FLPR handles higher layers of the software stack.
+
+.. _nrf54l_hpf_softperi_comparison_requirements:
+
+Requirements
+************
+
+For Soft Peripherals, there is a specific memory size requirement of approximately 16K, but there is a flexibility in placement within the FLPR execution RAM.
+To enter low power consumption modes, Soft Peripherals require a slot at a specific memory address, which is platform-specific, and the application code must be able to access MEMCONF registers.
+You can check these requirements in the :ref:`Soft Peripherals section<nrfxlib:soft_peripherals>`, which provide essential information for proper integration and optimization.
+
+On the other hand, HPF necessitates that the application core uses the Zephyr operating system.
+The framework allows for a high degree of configurability, which you can tailor according to your needs using Kconfig and devicetree settings.
+This flexibility allows you to optimize the memory footprint and functionality of the custom peripherals, ensuring that the peripherals are well-integrated and perform efficiently within their specific application environments.
+
+.. _nrf54l_hpf_softperi_comparison_features:
+
+Features comparison
+*******************
+
+See the following detailed feature comparison between Soft Peripherals and High-Performance Framework:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
+     - Soft Peripherals
+     - High Performance Framework
+   * - Integration
+     - Pre-compiled binary for FLPR, driver for Application Core
+     - Built from source in Zephyr build system
+   * - Memory requirements
+     - Fixed (~16K)
+     - Adjustable, depends on code and devicetree
+   * - Application Compatibility
+     - Can be used on baremetal or Zephyr applications
+     - Uses Zephyr without kernel on FLPR side, full Zephyr on APP side
+   * - Integration complexity
+     - Minor integration required
+     - Samples provided for the start of development
+   * - IPC mechanism
+     - Register-based
+     - ic(b)msg or mbox-based
+   * - Compliance & testing
+     - Protocol-compliant, verified by Nordic Semiconductor
+     - Samples provided as-is, testing is done by the user
+   * - Delivery form
+     - Binary
+     - Source code, modifiable and extendable
+   * - API level
+     - Exposed at HW-driver level
+     - FLPR may handle parts of SW stack above HW-driver level
+
+.. _nrf54l_hpf_softperi_comparison_creating_peripherals:
+
+Creating software-based peripherals
+***********************************
```

Comment:
Gerund is commonly used in our docs to indicate user-actionable step procedure. This conceptual documentation does not provide such steps, so this should be rephrased not to suggest them.